Dockerfile

```diff
@@ -41,9 +41,8 @@ RUN test -d ${REMOTE_SOURCES}/cachito-gomod-with-deps || exit 0; \
     cp -f $REMOTE_SOURCES_DIR/cachito-gomod-with-deps/app/registry-ca.pem . \
  && cp -f $REMOTE_SOURCES_DIR/cachito-gomod-with-deps/app/frontend/{.npmrc,.yarnrc,yarn.lock} frontend/
 
-# prevent download of chromedriver, geckodriver, and the cypress binary as part of module installs
-ENV CHROMEDRIVER_SKIP_DOWNLOAD=true \
-    GECKODRIVER_SKIP_DOWNLOAD=true \
+# prevent download of geckodriver, and the cypress binary as part of module installs
+ENV GECKODRIVER_SKIP_DOWNLOAD=true \
```

Comment:
I believe `GECKODRIVER_SKIP_DOWNLOAD=true` can be removed as well as that applied to Protractor.